src/dfcx_scrapi/tools/analysis_util.py

```diff
@@ -0,0 +1,162 @@
+"""Util class for performing analysis on DFCX objects and data."""
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import pandas
+import numpy as np
+from dfcx_scrapi.core.intents import Intents
+import google.cloud.dialogflowcx_v3beta1.types as types
+
+# logging config
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s %(levelname)-8s %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S')
+
+class AnalysisUtils():
```

Comment:
@Omerside update `AnalysisUtils` to be singular `AnalysisUtil`

We'll use this for naming standard convention for any file using `*_util.py` or class using `*Util` going forward.